config/settings.yml

```diff
@@ -1,5 +1,4 @@
 :api:
   :token_ttl: 10.minutes
   :authentication_timeout: 30.seconds
-  :metrics_default_limit: 1000
-  :event_streams_default_limit: 1000
```

Comment:
If you are removing settings, you should probably create a data migration against the settings_changes in case someone has modified these values previously.

Reply:
@Fryguy AFAIK neither of these settings have made it into an official release yet - does that change anything?